src/megachatapi_impl.cpp

```diff
@@ -5049,8 +5068,15 @@ MegaChatSessionPrivate::MegaChatSessionPrivate(const rtcModule::ISession &sessio
 }
 
 MegaChatSessionPrivate::MegaChatSessionPrivate(const MegaChatSessionPrivate &session)
-    : state(session.getStatus()), peerid(session.getPeerid()), clientid(session.getClientid()), av(session.hasAudio(), session.hasVideo()),
-      networkQuality(session.getNetworkQuality()), audioDetected(session.getAudioDetected())
+    : state(session.getStatus())
+    , peerid(session.getPeerid())
```

Comment:
I've added the comma at beginning to avoid that if we add a new member initialization the PR diff code shows changes in two lines, similar to enumeration with adding the last comma